src/views/workflow-history/helpers/check-history-event-group/is-activity-event.ts

```diff
@@ -0,0 +1,16 @@
+import type { HistoryEvent } from '@/__generated__/proto-ts/uber/cadence/api/v1/HistoryEvent';
+
+import type { ActivityHistoryEvent } from '../../workflow-history.types';
+
+export default function isActivityEvent(
+  event: Pick<HistoryEvent, 'attributes'>
+): event is ActivityHistoryEvent {
+  return [
+    'activityTaskScheduledEventAttributes',
```

Comment:
Instead of hardcoding them here, it would be great if you could use constants for these. And potentially, even better if you could use generics to create a single util.

Maybe something like this:
`function isEvent<EventType extends GeneratedTypeFromCadence>(eventValues, event): event is EventType`